src/world/world.lua

```diff
@@ -18,6 +19,8 @@ function World:__create(playerHostility)
 	self.physics:setCallbacks(World.beginContact, World.endContact,
 							  World.preSolve, World.postSolve)
 
+	self.screen = Screen()
```

Comment:
I think this should be in ingame or initworld, because I would like as little ui code in/beneath world.lua as possible. I look at world as the physics management code.